v0/util/cache.js

```diff
@@ -17,12 +17,26 @@ class Cache {
 
     return storeFunction().then(result => {
```

Comment:
In that case, it will fail. But it is structured in that way only like if the value is not in the cache then this function is being used with some implementations to get the value and store it.